azure-quantum/environment.yml

```diff
@@ -23,3 +23,4 @@ dependencies:
     - qiskit-terra>=0.18.0
     - asyncmock>=0.4.0
     - aiofile>=3.7.2
+    - protobuf==3.14.0
```

Comment:
wonder if this should be an optional dependency

Reply:
Hmm, I have added some code that takes dependency on google protobuf functions. That code breaks without having this available.